alf/environments/suite_socialbot.py

```diff
@@ -33,6 +33,43 @@
 DEFAULT_SOCIALBOT_PORT = 11345
 
 
+class UnwrappedEnvChecker(object):
```

Comment:
Should be in environments/utils.py

Reply:
That will create a circular dependency: suite_socialbot -> utils -> suite_socialbot

Reply:
changed